packages/peregrine/lib/util/deriveErrorMessage.js

```diff
@@ -7,9 +7,7 @@
 const toString = error => {
     const { graphQLErrors, message } = error;
 
-    return graphQLErrors && graphQLErrors.length
-        ? graphQLErrors.map(({ message }) => message).join(', ')
-        : message;
+    return graphQLErrors && graphQLErrors.length ? 'graphQLErrors' : message;
```

Comment:
Unfortunately there are a handful of other places besides `FormError` that use this function, and they will break with this change.

Since the goal of this ticket is to _not_ show server errors to end users, and `deriveErrorMessage` currently does exactly that, what would you think about replacing all usages of `deriveErrorMessage` to return the `formError.errorMessage` (perhaps rename it to `global.errorMessage`)?

We should still `console.warn` the previous output of this function (`graphQLErrors.map(({ message }) => message).join(', ')`) if we have `graphQLErrors`.